src/Frontend/Core.hs

```diff
@@ -303,18 +304,35 @@ instance (Typeable a) => ToHtml (AuthorWidget a) where
             span_ [class_ "author-image"] $ avatarImgFromMeta mi
             span_ [class_ "author-text"] $ mi ^. metaCreatedByLogin . fromUserLogin . html
 
+data ListItemIdeaContext
+    = IdeaInIdeasOverview
+    | IdeaInViewTopic
+    | IdeaInUserProfile
+  deriving (Eq, Show, Read)
+
 data ListItemIdea = ListItemIdea
-      { _listItemIdeaLinkToUser :: Bool
+      { _listItemIdeaContext    :: ListItemIdeaContext
       , _listItemIdeaPhase      :: Maybe Phase
       , _listItemIdeaNumVoters  :: Int
       , _listItemIdea           :: Idea
+      , _listItemUserRole       :: Role
       }
   deriving (Eq, Show, Read)
 
 instance ToHtml ListItemIdea where
     toHtmlRaw = toHtml
-    toHtml p@(ListItemIdea _linkToUserProfile _phase numVoters idea) = semanticDiv p $ do
+    toHtml p@(ListItemIdea listItemIdeaContext phase numVoters idea role) = semanticDiv p $ do
         div_ [class_ "ideas-list-item"] $ do
+            -- FIXME: Visibility, translation, already marked
+            when isIdeaInJury . div_ $ do
+                case _ideaJuryResult idea of
+                    Nothing -> do
+                        button_ [onclick_ $ P.juryIdea idea IdeaFeasible]    "FEASIBLE"
```

Comment:
as a rule, capitalize first letter and make everything else small for buttons etc.  css will handle the details.